Framework/CurveFitting/src/Functions/ConvTempCorrection.cpp

```diff
@@ -0,0 +1,41 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI,
+//   NScD Oak Ridge National Laboratory, European Spallation Source,
+//   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+// SPDX - License - Identifier: GPL - 3.0 +
+
+#include "MantidCurveFitting/Functions/ConvTempCorrection.h"
+#include "MantidAPI/FunctionFactory.h"
+#include <cmath>
+
+namespace {
+constexpr double CONVERSIONFACTOR = 11.606;
```

Comment:
We need a comment explaining what this value is / how it was derived or a citation

Reply:
Ill try get hold of some sort of reference tomorrow, I was mostly just copy and pasting from the original declaration, but there must be a source somewhere.

Reply:
its probably 1/k  in funny units (e.g. 0.695 cm^{-1}/K)